proto/config.json

```diff
@@ -0,0 +1,35 @@
+{
+  "version": "0.1.0",
+  "platform": {
+    "os": "linux",
+    "arch": "amd64"
+  },
+  "process": {
+    "terminal": true,
+    "user": {
+      "@type": "type.googleapis.com/oci.LinuxUser",
```

Comment:
On Wed, Sep 30, 2015 at 10:29:33AM -0700, Timothy Chen wrote:
> > +      "@type": "type.googleapis.com/oci.LinuxUser",
> 
> Is type added by the json generation?

It's added by protobuf so it can figure out which message type to use
for deserializing Any fields.  See [1].

[1]: https://developers.google.com/protocol-buffers/docs/proto3#json